mis_builder_subkpi_template/models/mis_report_instance.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+from odoo import models, fields, api
+
+
+class MisReportInstance(models.Model):
+
+    _inherit = 'mis.report.instance'
+
+    report_subkpis = fields.Boolean(
+        compute='_compute_report_subkpis',
+        store=True,
+    )
+    subkpis_filter_active = fields.Boolean(
+        string='Sub-KPIs filter active',
+        default=False,
+    )
+    filter_subkpi_ids = fields.Many2many(
+        'mis.report.subkpi',
+        string='Sub-KPIs to display',
+    )
```

Comment:
How is this different from the `subkpi_ids` field on `mis.report.instance.period`.
Is it the goal to simplify configuration by filtering subkpis at once for all columns?
In this case, it's best to add this feature in the core module.

Reply:
It is the same idea you are right.
In our case, subkpis = department so our accountant want to be able to display a P&L with department in columns then filter quickly from the report to a specific department. 

If we have your blessing, we can add this feature in the core module, yes :)

Reply:
@sbidoul Yes it is, so I will extract the feature and open another PR to target the core module.